### PR TITLE
docs: Adjust the ordering of the sections

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Review by spouliot — APPROVED** [review]

File: doc/articles/common-issues-all-ides.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ uid: Uno.UI.CommonIssues.AllIDEs
 
 # Issues related to all development environments
 
+## Could not resolve SDK "Uno.Sdk"
+
+This error may happen for multiple reasons:
+
+- Ensure that all [nuget feeds are authenticated properly](https://learn.microsoft.com/en-us/nuget/consume-packages/consuming-packages-authenticated-feeds). When building on the command line, some enterprise nuget feeds may not be authenticated properly.
```
Comment (Youssef1313) [inline doc/articles/common-issues-all-ides.md:11]:
```suggestion
- Ensure that all [NuGet feeds are authenticated properly](https://learn.microsoft.com/nuget/consume-packages/consuming-packages-authenticated-feeds). When building on the command line, some enterprise nuget feeds may not be authenticated properly.
```

File: doc/articles/common-issues-all-ides.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ uid: Uno.UI.CommonIssues.AllIDEs
...
+- Ensure that no global package mappings are interfering with nuget restore. To validate that no package mappings are set, on Windows for Visual Studio 2022:
+  - Make a backup copy of `%appdata%\NuGet\NuGet.Config`
```
Comment (Youssef1313) [inline doc/articles/common-issues-all-ides.md:13]:
```suggestion
  - Make a backup copy of `%AppData%\NuGet\NuGet.Config`
```

File: doc/articles/common-issues-vs2022.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ uid: Uno.UI.CommonIssues.vs2022
 
 # Issues related to Visual Studio 2022 for Windows
 
+## Unable to select the `MyApp (Unpackaged WinAppSDK)` profile
+
+A [Visual Studio issue](https://developercommunity.visualstudio.com/t/WinAppSDK-Unpackaged-profile-cannot-be-s/10643735) is preventing the Unpackaged profile if iOS/Android target frameworks are present in the project. In order for the unpackaged profile to be selected, you'll need to edit the `Properties/launchSettings.json` file to remove the `MyApp (Packaged WinAppSDK)` entry. Once it is removed, select the `MyApp (Unpackaged WinAppSDK)` the start the debugging of your app.
```
Comment (Youssef1313) [inline doc/articles/common-issues-vs2022.md:9]: > select the `MyApp (Unpackaged WinAppSDK)` the start the debugging of your app.

There is a typo here I think? The second "the" is probably meant as "then" or "to"?

**Review by Youssef1313 — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**